react/ActionMenu/Readme.md

```diff
@@ -116,39 +116,42 @@ The ActionMenu component is rendered at the root of the DOM tree using a [Portal
 In that case, we can use the `containerElRef` prop to provide a reference to an element where ActionMenu will be rendered. Here is an example for the most common case, an ActionMenu inside a Modal:
```

Comment:
I think we need to update this description text as we no longer use the prop in the example. Maybe we can also update the proptype description for `containerRefEl`?